draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1079,15 +1079,32 @@ Decoy digests increase the size of the SD-JWT. The number of decoy digests (or w
 
 ## Unlinkability
 
-Colluding Issuer/Verifier or Verifier/Verifier pairs could link issuance/presentation
-or two presentation sessions to the same user on the basis of unique values encoded in the SD-JWT
-(Issuer signature, salts, digests, etc.).
-
-To prevent these types of linkability, various methods, including but not limited to the following ones can be used:
-
-- Use advanced cryptographic schemes, outside the scope of this specification.
-- Issue a batch of SD-JWTs to the Holder to enable the Holder to use a unique SD-JWT per Verifier. This only helps with Verifier/Verifier unlinkability.
-
+Unlinkability is a property that prevents an adversary from associating two or
+more sessions of the same user. Two types of unlinkability are often considered:
+
+- Verifier/Verifier unlinkability: Two colluding Verifiers should not be able to
+  tell that they have received presentations from the same user.
+- Issuer/Verifier unlinkability: An Issuer colluding with a Verifier should
+  not be able to tell that a user that has received a credential from the Issuer
+  has presented it to the Verifier.
+
+In both cases, unlinkability is limited to cases where the
+disclosed claims do not contain information that directly or indirectly
```

Comment:
Really? There can be multiple same people with the same birthday. In which case, if the holder discloses only the birthdate to the verifier, the issuer/verifier unlinkability is possible.

Reply:
Yes, but in that case, the information was obviously not enough to identify the user.

Reply:
I think your argument is that the example I give above still does not prevent issuer/verifier unlinkability in sd-jwt because the user's public key becomes the correlation factor, which I think needs to be explicitly stated

Reply:
No, what I want to say is: The amount of information that can identify a user varies depending on context knowledge and the set of users that exist. If I'm looking at a club with 12 members all having a different birthdates, the birthdate is sufficient identifying information. If there would be a specific credential for sitting US presidents, unlinkability cannot be achieved at all, even if only the credential type is shared.

So the point is: For batch issuance, even if all keys and so on are fresh for each credential, there is an upper limit on the grade of unlinkability achievable, and it depends on the dataset shared and context knowledge etc.